projects/admin.py

```diff
@@ -125,3 +129,71 @@ class DocumentTemplateAdmin(admin.ModelAdmin):
 @admin.register(ProjectAttributeImage)
 class ProjectAttributeImageAdmin(admin.ModelAdmin):
     pass
+
+
+@admin.register(PhaseAttributeMatrixStructure)
+class PhaseAttributeMatrixStructureAdmin(admin.ModelAdmin):
+    change_form_template = "admin/matrix.html"
+
+    def render_change_form(
+        self, request, context, add=False, change=False, form_url="", obj=None
+    ):
+        cell_rows = []
+        if obj:
+            # The matrix is square so we just need one side to know the other
+            side_length = len(obj.column_names)
+
+            # Fill matrix with None values
+            cell_rows = [
+                [None for x in range(0, side_length)] for y in range(0, side_length)
```

Comment:
`[None] * side_length` should be equal to `[None for x in range(0, side_length)]`

Reply:
Also `list(range(0,5)) == list(range(5))`

Reply:
So the short version of this would be: `[[None] * side_length] * side_length`